rest-service/manager_rest/resource_manager.py

```diff
@@ -56,8 +56,11 @@
                                 extract_host_agent_plugins_from_plan)
 from manager_rest.rest.rest_utils import (
     RecursiveDeploymentDependencies,
-    update_inter_deployment_dependencies, verify_blueprint_uploaded_state,
-    compute_rule_from_scheduling_params)
+    RecursiveDeploymentLabelsDependencies,
+    update_inter_deployment_dependencies,
+    verify_blueprint_uploaded_state,
+    compute_rule_from_scheduling_params
```

Comment:
A trailing comma would be nice here, but not critical at all.

Reply:
done